project/dimensions/enduses.csv

```diff
@@ -1,14 +1,15 @@
 id,name,fuel_id,unit
 electricity_computers_and_tvs,Computers and TVs,electricity,MWh
 electricity_dishwasher,Dishwasher,electricity,MWh
+electricity_clothes_washing,Clothes Washing,electricity,MWh
 electricity_drying,Drying,electricity,MWh
 electricity_cooking,Cooking,electricity,MWh
-electricity_cooling_and_heating,Cooling and Heating,electricity,MWh
+electricity_cooling_and_heating,Heating and Cooling,electricity,MWh
 electricity_fans_and_pumps,Fans and Pumps,electricity,MWh
 electricity_lighting,Lighting,electricity,MWh
-electricity_miscellaneous_electrical_loads,Miscellaneous Electrical Loads (MELs),electricity,MWh
-electricity_non-PC_office_equipment,Non-PC Office Equipment,electricity,MWh
-electricity_other,Other,electricity,MWh
+electricity_miscellaneous_electrical_loads,Commercial Miscellaneous Electrical Loads (MELs) & Other,electricity,MWh
+electricity_non-PC_office_equipment,Office Equipment,electricity,MWh
```

Comment:
@elainethale Any object to `electricity_non_pc_office_equipment` instead?

Reply:
No objection from me. I think I was just retaining some of the punctuation from the Scout documentation. @asatremeloy, any concerns here?

Reply:
No concerns on my end. "Non-PC" is technically correct since that is an AEO/NEMS technology type for commercial buildings and since we have a separate end-use for "Computers and TVs", but I think in either case it should be clear what we're covering that end-use category.

Reply:
Our record IDs are typically in the format of `snake_case` for predictability reasons. The name column can be the user-preferred display name. In dsgrid queries the user has the option to display either value.

Reply:
Sounds good, this is fine with me.